kong/information.go

```diff
@@ -0,0 +1,53 @@
+package kong
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+)
+
+type Info struct {
+	Version       string `json:"version,omitempty" yaml:"version,omitempty"`
+	Configuration struct {
+		Database string `json:"database,omitempty" yaml:"database,omitempty"`
+		Portal   bool   `json:"portal,omitempty" yaml:"portal,omitempty"`
+		RBAC     string `json:"rbac,omitempty" yaml:"rbac,omitempty"`
+	} `json:"configuration,omitempty" yaml:"configuration,omitempty"`
+}
+
+// IsEnterprise check if Kong is enterprise edition
+func (s *Info) IsEnterprise() bool {
+	return strings.Contains(s.Version, "enterprise")
+}
+
+// IsInMemory check if Kong is in memory
+func (s *Info) IsInMemory() bool {
+	return "off" == s.Configuration.Database
+}
+
+// IsRBACEnabled check if RBAC are enabled
+func (s *Info) IsRBACEnabled() bool {
+	return "on" == s.Configuration.RBAC
+}
```

Comment:
Please implement these on the `Configuration` object directly. (Will probably need un-inlining the `Configuration` `struct` definition).

Reply:
There is a conflict with another Configuration, here
https://github.com/Kong/go-kong/blob/402cf0f5dd1df3f56059fb3f152bbd1c938b6a84/kong/types.go#L237

What do you suggest?

Reply:
Let's go with `RuntimeConfiguration`.